app/controllers/users_controller.rb

```diff
@@ -1,8 +1,8 @@
 class UsersController < ApplicationController
-  
-  def follow
-  end
-
-  def unfollow
+  def show
+    @user = User.find(params[:id])
+    @actual_user = current_user
+    @rel = @user.followers.find_by(follower: current_user)
+    @actual_current_user = true if @user.id == current_user.id
```

Comment:
this could also be a method

Reply:
All changes were applied in the following commits of this same PR